### PR TITLE
More citizen names 1.16

### PR DESCRIPTION
**Pull request by ravenbuilder934**
Closes https://github.com/ldtteam/minecolonies-features/issues/218
Closes #
Closes #

# Changes proposed in this pull request:
- Adds more citizen names
- Same as #6252 
-

Review please

### CODE REVIEW

File: src/api/java/com/minecolonies/api/util/constant/NameConstants.java [hunk 1]
```diff
@@ -6,592 +6,157 @@
...
-      "Roger",
-      "Rowland",
-      "Simon",
-      "Solomon",
-      "Stephen",
-      "Tobias",
-      "Walter",
-      "William",
-      "Kysen"
+      "Aarav", "Aaron", "Aayan", "Abdiel", "Alain", "Aaliyah", "Abdullah", "Abel", "Abraham", "Abram", "Ace", "Adam", "Adan", "Aden",  "Adonis", "Adrian", "Adriel", "Adrien",
```
Comment (Asherslab) [inline src/api/java/com/minecolonies/api/util/constant/NameConstants.java:9]: you broke the formatting! =P

Comment (ravenbuilder934) [reply src/api/java/com/minecolonies/api/util/constant/NameConstants.java:9]: If it doesn't need to be on a new line I'm not making a new line for _over a thousand_ names

Comment (Asherslab) [reply src/api/java/com/minecolonies/api/util/constant/NameConstants.java:9]: blasphemy

Comment (Asherslab) [reply src/api/java/com/minecolonies/api/util/constant/NameConstants.java:9]: also it's automatic if you use an IDE not GITHUB =P

Comment (ravenbuilder934) [reply src/api/java/com/minecolonies/api/util/constant/NameConstants.java:9]: Okay, give me a few hours to set up an ide 🙃

Comment (ravenbuilder934) [reply src/api/java/com/minecolonies/api/util/constant/NameConstants.java:9]: I installed IntelliJ and pulled up the changed file, how do I reformat it?

Comment (Raycoms) [reply src/api/java/com/minecolonies/api/util/constant/NameConstants.java:9]: @Asherslab

Comment (Asherslab) [reply src/api/java/com/minecolonies/api/util/constant/NameConstants.java:9]: i was joking tbh

**Review by Asherslab — APPROVED** [review]
because it wasn't clear lol

**Comment by Raycoms** [comment]
:shipit: 

[Approved with PullApprove](https://pullapprove.com/ldtteam/minecolonies/pull-request/6251/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=ldtteam/minecolonies)

**Pull request merged**